src/config/config.js

```diff
@@ -221,6 +221,18 @@ const config = convict({
     nullable: true,
     default: null,
     env: 'CDP_HTTPS_PROXY'
+  },
+  auditingEnabled: {
+    doc: 'Enable sending audit events to the CDP auditing stack.',
+    format: Boolean,
+    default: process.env.NODE_ENV === 'production',
```

Comment:
Will this also fire in a docker compose environment?

Reply:
Do we need to enable firehose on localstack?

Reply:
yeap, tested it locally, we just need to override the endpoint on the client for it to work